clang/include/clang/Sema/Sema.h

```diff
@@ -11314,6 +11314,11 @@ class Sema final {
   void
   checkDeclIsAllowedInOpenMPTarget(Expr *E, Decl *D,
                                    SourceLocation IdLoc = SourceLocation());
+
+  /// Adds OMPDeclareTargetDeclAttr to referenced variables in declare target
```

Comment:
Instead of explaining what the function does _literally,_ a comment describing the goal of the function would be easier to understand.